released/packages/coq-huffman/coq-huffman.8.9.0/opam

```diff
@@ -1,39 +1,37 @@
 opam-version: "2.0"
```

Comment:
This package looks suspicious to me as it seems to conflict with an existing package. In particular there is a difference on the `date` tag.

Reply:
Indeed, this is an error. I now removed huffman from the list.